### ISSUE TITLE
"Phonemes" outdated 

### ISSUE THREAD

**Issue by Arkoniak** [FluxML/model-zoo#24]
Due to this change in Flux: https://github.com/FluxML/Flux.jl/commit/0f1e7b55780723ec1ecc282ba98b506cbf1480fe?diff=unified, "phonemes" model is no longer working, since it uses "combine" function that is no longer exists: https://github.com/FluxML/model-zoo/blob/master/phonemes/1-model.jl#L3

**Comment by abcsds** [comment]
I cannot reproduce this error with v0.4.1. I got my model training smoothly. I do not see where the combine function is comming from though. It's not being exported, but I have no problem importing it. Why could this happen? Is there maybe a cached pre-compiled?
```julia
julia> Pkg.status("Flux")
 - Flux                          0.4.1

julia> using Flux: combine
INFO: Recompiling stale cache file ~/.julia/lib/v0.6/Flux.ji for module Flux.

julia> combine([2,3,4],[1,2,3])
6-element Array{Int64,1}:
 2
 3
 4
 1
 2
 3
```

**Comment by Arkoniak** [comment]
Yes, you are right. It turns out I am using master branch of Flux. I am not sure what to do with this issue - I can close it, but if new version of Flux is going to be tagged without `combine` function, this issue will rise again.

**Comment by amckay1** [comment]
Seems to have popped back up with version 0.5.1, and switching to master did not solve the problem.

**Comment by dpsanders** [comment]
Bump

**Comment by MikeInnes** [comment]
Fixed on the julia-0.7 branch (soon to be merged).
